ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml

```diff
@@ -23,18 +23,19 @@
 @if (Model.ProfileUser.IsBanned)
 {
     <div class="ui inverted red segment">
-        <h2>User is currently banned!</h2>
         @if (Model.User != null && Model.User.IsModerator)
         {
+            <h3 style="margin-bottom:3px;"><i class="ban icon"></i> This user is currently banned.</h3>
```

Comment:
This and the header on line 37 can probably be moved above the if statement on line 26